hw03_adkinsh2.py

```diff
@@ -0,0 +1,84 @@
+#################################################################################
+# Author: Harry Adkins
+# Username: adkinsh2
+#
+# Assignment: HW03
+# Purpose: Drawing stars
+# Google Doc Link: https://docs.google.com/document/d/1DqjcrDnNRH_vvrzYEIaPAd95IWsKW53YRXWLzz5ADKw/edit
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle
+
+
+def drawStar():
+    """
+    Example docstring for function_1. function_1 is not a good function name and should be changed.
+    """
+    bigStar = turtle.Turtle()
+    bigStar.pensize(4)
+    bigStar.color("white")
+
+    bigStar.penup()
+    bigStar.left(60)
+    bigStar.forward(200)
+    bigStar.right(60)
+    bigStar.pendown()
+
+    for i in range(6):
+        for j in range(5):
+            bigStar.forward(30)
+            bigStar.right(144)
+        bigStar.penup()
+        bigStar.right(60)
+        bigStar.forward(100)
+        bigStar.pendown()
+
+
+
+    pass
+    # ....
```

Comment:
Remove template artifacts such as these two lines, and extra excessive whitespace as well.